Service/EndpointService.php

```diff
@@ -143,6 +143,7 @@ public function handleRequest(Request $request): Response
             foreach ($endpoint->getThrows() as $throw) {
                 $event = new ActionEvent('commongateway.action.event', $parameters, $throw);
                 $this->eventDispatcher->dispatch($event, 'commongateway.action.event');
+                $parameters['response'] = $event->getData()['response'];
             }
 
             $parameters['response'] = $event->getData()['response'];
```

Comment:
Doesn't the code always take the last event response? So is the added code needed?

Reply:
No the code didn't used the last event response because we do not update the parameters['response'] value in the foreach